packages/flutter/test/cupertino/route_test.dart

```diff
@@ -2432,53 +2594,6 @@ void main() {
       expect(tester.getBottomRight(find.byType(Placeholder)).dx, 390.0);
     });
   });
-
-  testWidgets(
-  // TODO(polina-c): remove when fixed https://github.com/flutter/flutter/issues/145600 [leak-tracking-opt-in]
-  experimentalLeakTesting: LeakTesting.settings.withTracked(classes: <String>['CurvedAnimation']),
-  'Fullscreen route does not leak CurveAnimation', (WidgetTester tester) async {
```

Comment:
What is the reason to remove this test?

Reply:
Oops sorry for that. This must have come from the merge conflicts. I added it back in [test: Add back missing test](https://github.com/flutter/flutter/pull/146999/commits/7a2129a9537bcda7be5fdb28fd36c887ffd84310)